components/script_bindings/codegen/codegen.py

```diff
@@ -980,16 +980,17 @@ def fromJSValTemplate(config: str, errorHandler: str, exceptionCode: str) -> str
             unwrapFailureCode = failureCode
 
         templateBody = fill(
-            """
-            match ${function}($${val}, *cx) {
-                Ok(val) => val,
-                Err(()) => {
-                    $*{failureCode}
-                }
+        """
+        match ${function}($${val}, cx) {
+            Ok(val) => val,
+            Err(()) => {
+                $*{failureCode}
             }
-            """,
-            failureCode=unwrapFailureCode + "\n",
-            function=conversionFunction)
+        }
+        """,
+        failureCode=unwrapFailureCode + "\n",
+        function=conversionFunction)
```

Comment:
Hrm. Why is the indent changing here? This part of the diff might go away when running `./mach fmt`.

Reply:
I went ahead and added a commit that undoes this change; I don't think we have any python automatic formatting.